include/unk_02026E30.h

```diff
@@ -1,9 +1,9 @@
 #ifndef POKEHEARTGOLD_UNK_02026E30_H
 #define POKEHEARTGOLD_UNK_02026E30_H
 
-void sub_02026E30(void);
+void Init3DBufferSwapRequestState(void);
```

Comment:
I would prefer prefixes for these  functions, but I suppose there's also nothing obvious that can be put here so it's fine